src/main/java/com/amazon/opendistroforelasticsearch/security/ssl/transport/OpenDistroSecuritySSLRequestHandler.java

```diff
@@ -81,6 +82,11 @@ public final void messageReceived(T request, TransportChannel channel, Task task
             channel.sendResponse(exception);
             throw exception;
         }
+
+        String channelType = channel.getChannelType();
+        if (!channelType.equals("direct") && !channelType.equals("transport")) {
```

Comment:
Wasn't this originally - (!"netty".equals(channel.getChannelType()) && !"direct".equals(channel.getChannelType())) based on https://github.com/opendistro-for-elasticsearch/security-ssl/pull/2/files.

Why are we checking for !channelType.equals("transport")?

Reply:
Maybe its changed from elastic side https://github.com/elastic/elasticsearch/pull/40074

Reply:
netty is renamed to transport

Reply:
Thanks for clarification